gme/gme.exports

```diff
@@ -24,6 +24,7 @@ gme_open_file
 gme_play
 gme_seek
 gme_seek_samples
+gme_seek_scaled
```

Comment:
Hm, maybe move this new addition (and the toher below) to the bottom of the list? (Like it was done for the new api additions in 0.6.4)